main.go

```diff
@@ -29,12 +29,20 @@ import (
 
 	clientset "github.com/xmudrii/etcdproxy-controller/pkg/client/clientset/versioned"
 	informers "github.com/xmudrii/etcdproxy-controller/pkg/client/informers/externalversions"
+	"github.com/xmudrii/etcdproxy-controller/pkg/controller/corecerts"
+	"github.com/xmudrii/etcdproxy-controller/pkg/controller/etcdproxy"
 	"github.com/xmudrii/etcdproxy-controller/pkg/signals"
 )
 
 var (
 	masterURL  string
 	kubeconfig string
+
+	etcdURL string
```

Comment:
Global variables will not serve you well in the long run. It doesn't need to be fixed here, but I recommend switching to cobra since that's effectively the kube standard.